src/js/utils/draw.js

```diff
@@ -98,13 +98,14 @@ export function wrapInSVGGroup(elements, className='') {
 	return g;
 }
 
-export function makePath(pathStr, className='', stroke='none', fill='none') {
+export function makePath(pathStr, className='', stroke='none', fill='none', strokeWidth=0) {
 	return createSVG('path', {
 		className: className,
 		d: pathStr,
 		styles: {
 			stroke: stroke,
-			fill: fill
+			fill: fill,
+			'stroke-width': strokeWidth
```

Comment:
I'd suggest we follow the camelCase across the code.

`strokeWidth: strokeWidth`